python/pysrc/kaskada/_execution.py

```diff
@@ -3,7 +3,7 @@
 from dataclasses import dataclass
 from typing import AsyncIterator, Callable, Iterator, Literal, Optional, TypeVar, Union
 
-import kaskada as kd
+import kaskada
```

Comment:
Hmm. Any reason we need to change the pattern from importing as `kd`? I thought that was our pretty much standard convention?